### DISCUSSION TITLE
ANURA3D COMPILING PROBLEM-SOLVED

### DISCUSSION THREAD

**Discussion by EmreDumlu-UM** [Anura3D/Anura3D_OpenSource#64] [General]
Dear All,

I have a similar problem with compiling the anura3d with Visual studio community 2019. Once I started to build the .sln file, errors appeared on the screen.

Is there anyone who knows to get through this?

Thanks

![vs error](https://user-images.githubusercontent.com/125078884/218162100-18690722-193c-4b5c-a487-0d04d73772b8.png)

![vs2](https://user-images.githubusercontent.com/125078884/218162517-074ea861-3089-4adc-b405-f54b703baadb.png)

**Comment by EmreDumlu-UM** [comment]
The problem caused by a bug is solved. The '&' term needs to be deleted at file MPMDynamicExplicit.FOR at the line 771.

**Reply by dujianc** [reply]
There is no line 771 in this subroutine!

**Comment by dujianc** [comment]
I can successfully compile the 2023 codes, but when run the executable, I get this error message: 
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/64575408/9faf277f-2d56-4a2d-ab40-1cc2e43062af)

**Reply by dujianc** [reply]
Can anybody help me out?

**Reply by murphy-john** [reply]
Will you explain more about what you're trying to do? I'm not familiar with the error you posted. There are some suggested guidelines for posting [here](https://github.com/Anura3D/Anura3D_OpenSource/discussions/29).

Did you put the executable in the folder where there input files (CPS_001 and GOM) are?
